src/riak_core_claimant.erl

```diff
@@ -84,6 +84,16 @@
 %%% API
 %%%===================================================================
 
+
+-ifdef(TEST).
+
+-export([stop/0]).
+
+stop() ->
```

Comment:
By only including this in test, you signal that this `stop()` function should better not be called from any other module. But, clearly, the handle_call is there and stop can therewith be triggered.

So the added value of the `ifdef(TEST)` is rather limited.